packaging/windows/sharedframework/bundle.wxs

```diff
@@ -26,33 +25,6 @@
     <Variable Name="DOTNETHOME" Type="string" Value="[ProgramFiles6432Folder]dotnet" bal:Overridable="no" />
     <Variable Name="BUNDLEMONIKER" Type="string" Value="$(var.ProductMoniker) ($(var.TargetArchitecture))" bal:Overridable="no" />
 
-    <Chain DisableSystemRestore="yes" ParallelCache="yes">
```

Comment:
This change is incorrect since you only need to remove the ExePackage entry for CRT Redist. The MSIPackage entries are required for bundling the host installers. With your current change, the bundle will not install anything :)